examples/train_classification_model.py

```diff
@@ -0,0 +1,199 @@
+"""Example of training Model."""
+
+import os
+from typing import cast, Dict, Any
+import numpy as np
+
+from pytorch_lightning import Trainer
+from pytorch_lightning.callbacks import EarlyStopping
+
+from pytorch_lightning.loggers import WandbLogger
+import torch
+from torch.optim.adam import Adam
+from torch.nn.functional import softmax
+
+from graphnet.training.loss_functions import CrossEntropyLoss
+from graphnet.data.constants import FEATURES, TRUTH
+from graphnet.data.sqlite.sqlite_selection import (
+    get_desired_event_numbers,
+)
+from graphnet.models import StandardModel
+from graphnet.models.detector.icecube import IceCubeDeepCore
+from graphnet.models.gnn import DynEdge
+from graphnet.models.graph_builders import KNNGraphBuilder
+from graphnet.models.task.classification import ClassificationTask
+from graphnet.training.callbacks import ProgressBar
+from graphnet.training.utils import (
+    get_predictions,
+    make_train_validation_dataloader,
+    save_results,
+)
+
+from graphnet.utilities.logging import get_logger
+
+logger = get_logger()
+
+# Configurations
+torch.multiprocessing.set_sharing_strategy("file_system")
+
+# Constants
+features = FEATURES.DEEPCORE
+truth = TRUTH.DEEPCORE[:-1]
+
+# Make sure W&B output directory exists
+WANDB_DIR = "./wandb/"
+os.makedirs(WANDB_DIR, exist_ok=True)
+
+# Initialise Weights & Biases (W&B) run
+wandb_logger = WandbLogger(
+    project="example-script",
+    entity="graphnet-team",
+    save_dir=WANDB_DIR,
+    log_model=True,
+)
+
+
+def train(config: Dict[str, Any]) -> None:
+    """Train model with configuration given by `config`."""
+    # Log configuration to W&B
+    wandb_logger.experiment.config.update(config)
+
+    # Common variables; equal distribution of pid.
+    train_selection = get_desired_event_numbers(
+        database=cast(str, config["db"]),
+        desired_size=cast(int, config["event_numbers"]),
+        fraction_noise=float(1 / 3),
+        fraction_muon=float(1 / 3),
+        fraction_nu_e=float(1 / 9),
+        fraction_nu_mu=float(1 / 9),
+        fraction_nu_tau=float(1 / 9),
+    )
+
+    logger.info(f"features: {features}")
+    logger.info(f"truth: {truth}")
+
+    (
+        training_dataloader,
+        validation_dataloader,
+    ) = make_train_validation_dataloader(
+        cast(str, config["db"]),
+        train_selection,
+        cast(str, config["pulsemap"]),
+        features,
+        truth,
+        batch_size=cast(int, config["batch_size"]),
+        num_workers=cast(int, config["num_workers"]),
+    )
```

Comment:
See comment regarding dataset config file.

Reply:
the script has been reconfigured to match the new config setup, but I need help setting up the datasets file.